client/src/components/_common/Button/Button.module.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
**⚠️ Misleading File Extension**
Unless you need to use SASS, please change the file extension to `.module.css`.

<details>

The file extension `.module.scss` is for using SASS. You should not need to use SASS.

</details>